### ISSUE TITLE
Move all hardcoded parameter values into a configuration file

### ISSUE THREAD

**Issue by koadman** [gjospin/PhyloSift#223]
Examples include `last` score thresholds, `cmalign --tau` and `--mxsize`, pplacer values, pretty much any number in the code besides 0 and 1. These should go into the phylosiftrc. We may need to add some further structure to phylosiftrc to maintain readability and compartmentalize the variable namespace.

**Comment by gjospin** [comment]
This is basically done.
I found a couple command line parameters that don't get used anymore:
- reverse_translate.  I believe this always gets done if needed.
- chunk_size.  This should be getting used but maybe when Aaron change the octopus, this got faded out.  This may no longer be needed as CHUNK_MAX_SEQS is accessible through the RC file.  

Also using a custom config file is no longer working out the way we intended it.  We check for all the common config files before we parse the options so that the CL options get priority over the rc files.  However if we specify a custom config file on the CL, then we have to parse that after the CL options that should have priority.  Any thoughts ? 

My personal preference would be to not use a custom config file since we already are reading the user specific RC file.
